src/current/v23.2/admission-control.md

```diff
@@ -24,21 +46,42 @@ Admission control can help if your cluster has degraded performance due to the f
 - The node has high CPU usage, visible in the **CPU percent** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#cpu-percent).
 - The node is experiencing out-of-memory errors, visible in the **Memory Usage** graph in the [**Hardware** dashboard]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#memory-usage). Even though admission control does not explicitly target controlling memory usage, it can reduce memory usage as a side effect of delaying the start of operation execution when the CPU is overloaded.
 
+## Operations subject to Admission Control
+
+Almost all database operations that use CPU or perform Storage IO are controlled by the admission control system. From a user's perspective, specific operations that are affected by admission control include:
+
+- [XXX](): ADD LINKS BACK AND FORTH TO/FROM THINGS
+- [XXX]() ASK SUMEER ABOUT WHETHER THIS LIST OF THINGS MAKES SENSE
+- [General SQL queries]({% link {{ page.version.version }}/selection-queries.md %}) have their CPU usage subject to admission control, as well as storage IO for writes to [leaseholder replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases).
+- [Bulk data imports]({% link {{ page.version.version }}/import-into.md %}).
+- [Backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}).
+- [Schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), including index and column backfills (on both the [leaseholder replica]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) and [follower replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft)).
+- [Deletes]({% link {{ page.version.version }}/delete-data.md %}) (including those initiated by [row-level TTL jobs]({% link {{ page.version.version }}/row-level-ttl.md %}))
```

Comment:
Also scans done by TTL are subject to CPU admission control

Reply:
I have added a clause to this line to note that info, PTAL and let me know if it seems reasonable